plugins/msg-broker/mcollective/lib/openshift/mcollective_application_container_proxy.rb

```diff
@@ -2375,7 +2389,16 @@ def rsync_destination_container(gear, destination_container, destination_distric
       reply = ResultIO.new
       source_container = gear.get_proxy
       platform = gear.group_instance.platform
+      source_used_blocks = get_quota(gear)
```

Comment:
I think it would make sense to assign `source_used_blocks` the value of `get_quota(gear)[1]` instead of `get_quota(gear)`.  It would be a little more prudent and explicit to do something like this:

```ruby
      quota = get_quota(gear)
      source_used_blocks = quota[1] unless quota.nil?
```

But I don't feel strongly about it.